rfc/rfc-63/rfc-63.md

```diff
@@ -0,0 +1,139 @@
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+# RFC-63: Hudi bundle standards
+
+## Proposers
+
+- @xushiyan
+
+## Approvers
+
+- @vinoth
+- @prasanna
+
+## Status
+
+JIRA: https://issues.apache.org/jira/browse/HUDI-3529
+
+## Abstract
+
+Hudi bundle jars are the user-facing artifacts that deserve special attention and careful reviews when changes happen on
+the bundle names, included jars, intended usage, etc. In this RFC, a group of standards about bundle building,
+dependency governance, and release are presented to serve as references for developers and users.
+
+## Building
+
+Bundles are built under [packaging/](../../packaging/) directory.
+
+| Bundle                      | Category  | Usage                                                                     |
+|-----------------------------|-----------|---------------------------------------------------------------------------|
+| hudi-spark-bundle           | Engine    | Integrates with Spark                                                     |
+| hudi-flink-bundle           | Engine    | Integrates with Flink                                                     |
+| hudi-hadoop-mr-bundle       | Engine    | Integrates with Hive                                                      |
+| hudi-presto-bundle          | Engine    | Integrates with Presto                                                    |
+| hudi-trino-bundle           | Engine    | Integrates with Trino                                                     |
+| hudi-kafka-connect-bundle   | Engine    | Integrates with Kafka Connect                                             |
+| hudi-aws-bundle             | Cloud     | Supports AWS-specific integration and features                            |
+| hudi-gcp-bundle             | Cloud     | Supports GCP-specific integration and features                            |
+| hudi-hive-sync-bundle       | Catalog   | Supports meta sync with Hive metastore                                    |
+| hudi-datahub-sync-bundle    | Catalog   | Supports meta sync with DataHub                                           |
+| hudi-timeline-server-bundle | Server    | Runs standalone timeline server                                           |
+| hudi-utilities-slim-bundle  | Utilities | Provides Hudi utilities to run with different engines                     |
+| hudi-utilities-bundle       | Utilities | Similar to "utilities-slim"; legacy bundle that contains Spark integration |
+| hudi-integ-test-bundle      | Test      | Runs with docker demo for integration tests                               |
+
+### Standard: engine integration
+
+An engine bundle should clearly declare which version it supports. If it's scala-dependent, it should also declare scala
+version in the jar name. The format is `hudi-{engine name}X.Y-bundle{_scala version}`. Examples:
```

Comment:
nit: `hudi-{engine name}X.Y-bundle_{scala version}`

Reply:
We should also call out what `X.Y` stand for exactly (major, minor versions). 

We should also call out that this binds us to stay compatible w/in a single minor branch (ie across all patch versions w/in the minor branch)